pkg/daemon/ceph/client/pool.go

```diff
@@ -265,9 +267,8 @@ func CreateReplicatedPoolForApp(context *clusterd.Context, clusterName string, n
 	}
 
 	// the pool is type replicated, set the size for the pool now that it's been created
-	sizeProperty := "size"
-	if err = SetPoolProperty(context, clusterName, newPool.Name, sizeProperty, strconv.FormatUint(uint64(newPool.Size), 10)); err != nil {
-		return errors.Wrapf(err, "failed to set property %q to replicated pool %q to %d", sizeProperty, newPool.Name, newPool.Size)
+	if err = SetPoolReplicatedSizeProperty(context, clusterName, newPool.Name, strconv.FormatUint(uint64(newPool.Size), 10), newPool.AllowUnsafeSingleReplica); err != nil {
```

Comment:
This new setting just applies to octopus, right? Do we not need to check the ceph version?

Reply:
Not really, it's a no-op for the CLI if the flag does not exist and I have forced `mon_allow_pool_size_one` in the operator ceph.conf file anyways.
So on the Rook side, we only check our CR spec, much easier than parsing the Ceph version.

Reply:
Ah perfect if nautilus just ignores the extra arg.